benchmarks.py

```diff
@@ -63,13 +64,7 @@ def run_test(test_name, iterations, avg_time_across_tests, completed_tests, rema
 
     return pass_count, fail_count, run_times
 
-def clear_lines(n=1):
-    """Clears a specified number of lines in the terminal."""
-    for _ in range(n):
-        sys.stdout.write('\x1b[1A')  # Move the cursor up by one line
-        sys.stdout.write('\x1b[2K')  # Clear the current line
-
-def print_summary_table(test_path, iterations, tests_results, total_run_time, output_dir):
+def print_summary_table(test_path, iterations, tests_results, total_run_time, output_dir, total_benchmarks):
```

Comment:
super nit: if we could add types here would be awesome hehe

Reply:
done